go/vt/schemadiff/schema.go

```diff
@@ -330,7 +331,7 @@ func (s *Schema) normalize() error {
 	// Validate table definitions
 	for _, t := range s.tables {
 		if err := t.validate(); err != nil {
-			return err
+			return errors.Join(errs, err)
```

Comment:
In the above case of `s.ValidateViewReferences()` we don't want to return all gathered errors up to that point?

Reply:
You're right, it's an oversight. Now both collecting the error along with all previous errors, _and_ also continuing the normalization process.